crates/re_ui/src/list_item2/property_content.rs

```diff
@@ -176,35 +175,67 @@ impl ListItemContent for PropertyContent<'_> {
             action_buttons,
         } = *self;
 
-        // We always reserve space for the action button(s), even if there are none.
-        let action_button_rect = egui::Rect::from_center_size(
-            context.rect.right_center() - egui::vec2(ReUi::small_icon_size().x / 2., 0.0),
-            ReUi::small_icon_size() + egui::vec2(1.0, 1.0), // padding is needed for the buttons
-        );
-
-        let content_width =
-            (context.rect.width() - action_button_rect.width() - ReUi::text_to_icon_padding())
-                .at_least(0.0);
-
-        //TODO(ab): adaptable columns
-        let column_width = ((content_width - ReUi::text_to_icon_padding()) / 2.).at_least(0.0);
+        // │                                                                              │
+        // │◀─────────────────────────────background_x_range─────────────────────────────▶│
+        // │                                                                              │
+        // │ ◀───────────state.left_column_width────────────▶│┌──COLUMN_SPACING           │
```

Comment:
Where is `context.rect` in this picture?

Reply:
It's actually `CONTENT`, but yeah it's better to use `context.rect` here 👍🏻